portfolio/src/main/webapp/style.css

```diff
@@ -3,7 +3,40 @@
   margin-right: auto;
   width: 650px;
 }
+body {
+  background-color: white;
+}
 
 #greeting-container {
   margin-top: 20px;
 }
+
```

Comment:
super nit: Try to be consistent with your whitespaces / newlines. In general, regardless of style, consistency is key. Some of these have new lines between, some don't.